esmvalcore/preprocessor/_regrid.py

```diff
@@ -360,15 +361,16 @@ def extract_location(cube, location, scheme):
     if scheme is None:
         raise ValueError("Interpolation scheme needs to be specified."
                          " Use either 'linear' or 'nearest'.")
-    geolocator = Nominatim(user_agent='esmvalcore')
+    ssl_context = ssl.create_default_context()
```

Comment:
thanks a lot @ehogan :beer: I agree with @bouweandela on this one; also, I was thinking maybe encase this in a `try/except` that, in case the ssl config creation fails (it's got a million and one reasons to fail, based on their docs), then it falls back to the previous implementation (no ssl changes) since we know that works 99% of the times (apart from the MO, that is)

Reply:
Thanks @bouweandela! I have done this in 80e3507d2 :)

Thanks @valeriupredoi! I have done this in 47cd6154b :)

Reply:
Thanks for making the changes @ehogan!